packages/demo/src/App.svelte

```diff
@@ -52,10 +52,10 @@
     </article>
   </section>
   <aside style="min-width: 319px;">
-    <sesamy-paywall
+    <!-- <sesamy-paywall
```

Comment:
Do we want this to be commented out?

Reply:
Yeah its just a second render of the same paywall but in a narrow sidebar, Kim did it to show the responsiveness. I can remove it